plugins/module_utils/vmware_guest.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
```

Comment:
If we'll put all the virtual devices related operations here, then the file name can be changed to vmware_device_helper.py or something other than "vmware_guest.py" to distinguish with the module name?

Reply:
Thanks, @mariolenz for the proposal!  
As you say, I think it is good that creates new module util as a common helper.

Though, as @Tomorrow9 says, I think it is better to change the file name from vmware_guest.

Reply:
@Tomorrow9 @sky-joker 
> If we'll put all the virtual devices related operations here, then the file name can be changed to vmware_device_helper.py or something other than "vmware_guest.py" to distinguish with the module name?

Good point. But I think `guest_device_helper.py` or `vm_device_helper.py`would be even better. To make it clear that this helper module is for guest (VM) devices and not for ESXi host devices.

Reply:
Sure, `vm_device_helper.py` is much better. Thanks.